run-test.sh

```diff
@@ -197,8 +197,7 @@ run_test()
     exit 0
   fi
 
-  dirName="$1/netcoreapp"
-
+  dirName="$1/netcoreapp-$OS-$ConfigurationGroup-$__Arch"
   if [ ! -d "$dirName" ]; then
```

Comment:
You should be able to remove this netstandard fallback logic.

Reply:
Thanks! I've update the PR.